app.js

```diff
@@ -155,6 +155,30 @@ app.post('/inventory/new_item', function(req, res) {
     })
 })
 
+// Inventory - Remove Item Route
+app.post('/inventory/remove_item', function (req, res) {
+
+    // Grab the necessary data from the POST request body
+    var product_name = req.body.product_name_input;
+    var wh_inventory = req.body.product_warehouse_inventory_input;
+    var shelf_inventory = req.body.product_shelf_inventory_input;
+
+    // Form the SQL Query needed to update the product inventory
+    var rem_inventory_query_string = "UPDATE products SET " +
+        "shelf_quantity = shelf_quantity - " + shelf_inventory + ", " +
+        "wh_quantity = wh_quantity - " + wh_inventory + " " +
+        "WHERE name = '" + product_name + "'"
```

Comment:
Just to be uniform with some of the changes I merged last night, what do you think about reformatting slightly to:

```
var rem_inventory_query_string = 
	`UPDATE products 
	SET shelf_quantity = shelf_quantity - ${shelf_inventory}, wh_quantity = wh_quantity - ${wh_inventory} 
	WHERE name = '${product_name}'`
```